content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
+This directive's location within the "Test" stage means that the environment variable `CI`, with its value of `true`, is only available within the scope of this "Test" stage.
+
+You can set this directive in Blue Ocean, so that its environment variable is available globally throughout Pipeline.
+This is also configured in the link:../build-a-node-js-and-react-app-with-npm/[Build a Node.js and React app with npm] tutorial.
+
+To do this:
+
+. From the main Blue Ocean interface, select *Branches* at the top-right to access your repository's `master` branch.
+. Select the `master` branch's "Edit Pipeline" icon image:tutorials/pipeline-in-blueocean-21a-edit-pipeline-on-branch.png[alt="Edit Pipeline on branch",width=3%] to open the Pipeline editor for this branch.
+. In the main Pipeline editor, select the *Test* stage you created <<add-a-test-stage-to-your-pipeline,above>> to begin editing it.
+. In the stage panel on the right, select *Settings* to reveal this section of the panel.
+. Select the minus (*-*) icon at the right of the `CI` environment directive you created earlier to delete it.
+. Select the top-left back arrow icon image:tutorials/pipeline-in-blueocean-16a-return-from-step-icon.png[alt="Return from step icon",width=3%] to return to the main Pipeline editor.
+. In the *Pipeline Settings* panel, select the *+* icon at the right of the *Environment* heading
+Here, you'll configure a _global_ environment directive.
+. In the *Name* and *Value* fields that appear, specify `CI` and `true`, respectively.
+. Select the *Save* button at the top-right to begin saving your Pipeline with with its relocated environment directive.
+. In the *Save Pipeline* dialog box, specify the commit message in the *Description* field, for example `Make environment directive global`.
+. Leaving all other options as is, select *Save & run*.
+Jenkins proceeds to build your amended Pipeline.
+. When the main Blue Ocean interface appears, select the _top_ row to see Jenkins build your Pipeline project.
++
+You should see the same build process as when you completed adding the final deliver stage (<<add-a-final-deliver-stage-to-your-pipeline,above>>).
+However, when you inspect the `Jenkinsfile` again, you'll notice that the `environment` directive is now a sibling of the `agent` section.
+
+== Wrapping up
+
+Well done! You've just used the Blue Ocean feature of Jenkins to build a simple Node.js and React application with npm!
+
+The "Build", "Test" and "Deliver" stages you created above are the basis for building other applications in Jenkins with any technology stack, including more complex applications and ones that combine multiple technology stacks together.
+
+Because Jenkins is extremely extensible, it can be modified and configured to handle practically any aspect of build orchestration and automation.
```

Comment:
```suggestion
Because Jenkins is extremely extensible, it can be modified and configured to handle almost any aspect of build orchestration and automation.
```